imdclient/tests/test_lammps.py

```diff
@@ -66,18 +67,4 @@ def true_u(self, topol, traj, imd_u, tmp_path):
 
     @pytest.fixture()
     def imd_u(self, docker_client, topol, tmp_path, port):
-        u = mda.Universe(
-            (tmp_path / topol),
-            f"imd://localhost:{port}",
-            atom_style="id type x y z",
-        )
-        with mda.Writer(
-            (tmp_path / "imd.trr").as_posix(), u.trajectory.n_atoms
-        ) as w:
-            for ts in u.trajectory:
-                w.write(u.atoms)
-        yield mda.Universe(
-            (tmp_path / topol),
-            (tmp_path / "imd.trr"),
-            atom_style="id type x y z",
-        )
+        u = minimalReader(f"imd://localhost:{port}", process_stream=True)
```

Comment:
It looks as if we are only using `process_stream=True` so that means that minimalReader should _always_ do this automatically. It's only purpose is to run inside the tests.